packages/vuetify/src/components/VList/VListItem.tsx

```diff
@@ -173,6 +176,10 @@ export const VListItem = genericComponent<VListItemSlots>()({
       if (!val) return
       handleActiveLink()
     })
+    watch(isActivated, val => {
+      if (!val || !scrollToActive) return
+      rootEl.value?.scrollIntoView({ block: 'nearest', behavior: 'smooth' })
```

Comment:
Last-minute check - I noticed (in Supabase) a Command Palette that did not have any smoothing when following the active item. Could you check if it looks more responsive/better without  it?